src/widgets/login-form/ui/LoginForm.tsx

```diff
@@ -35,13 +94,15 @@ export const LoginForm = () => {
               maxWidth={235}
               textAlign="center"
             >
-              <CustomButton
-                type="submit"
-                sx={loginButtonStyle}
-              >
+              <CustomButton type="submit" sx={loginButtonStyle}>
                 Войти
               </CustomButton>
-              <Link href="#" variant="body2" color="inherit" sx={linkStyle}>
+              <Link
+                href={Paths.Register}
+                variant="body2"
+                color="inherit"
+                sx={linkStyle}
+              >
                 Нет аккаунта? Зарегистрироваться
               </Link>
```

Comment:
Не тот компонент ссылки. Используй тот, что в `shared/ui/Link`. Цвет менять не надо, он по умолчанию чёрный.
```suggestion
              <Link
                href={Paths.Register}
                sx={linkStyle}
              >
                Нет аккаунта? Зарегистрироваться
              </Link>
```

Reply:
fixed